README.md

```diff
@@ -185,13 +186,14 @@ The above `insertAdditionalData()` method could then look like this:
     public function insertAdditionalData(Event $event) 
     {
         $data = [
-            ...
+           //your additional custom data 
         ];
 
+        //get modified_by_user_id from footprint plugin which holds the current logged in user entity
         if ($event->data('_footprint')) {
             $user = $event->data('_footprint');
-            $data += [
-                'user_id' => $user['id'],
+            $data += [                
+                'modified_by_user_id' => $user->id,
```

Comment:
im not sure about this, i use a custom auth and in $this->Auth->user() is an array with my oauth tokendata and the user entity.

Reply:
Why did you change the example?